cockroachDB/connection.js

```diff
@@ -0,0 +1,25 @@
+var pg = require("pg");
+var fs = require("fs");
+
+// ---------- Connect to the "booking" database.
+
+// ---------- NOTE this user does not have root permissions
+// ---------- 'booking' db must exist
+// ---------- db permissions must be granted to user
+
+var config = {
+  user: "maxroach",
+  host: "104.248.111.122",
+  database: "booking",
+  port: 26257,
+  ssl: {
+    ca: fs.readFileSync("./certs/ca.crt").toString(),
+    key: fs.readFileSync("./certs/client.maxroach.key").toString(),
+    cert: fs.readFileSync("./certs/client.maxroach.crt").toString()
+  }
+};
+
+// Create a pool.
+var pool = new pg.Pool(config);
+
+module.exports.pool = pool;
```

Comment:
This is from the cockroach docs, it opens a pg pool configured with the address of my load balancer on digital ocean

Reply:
I'm not sure about readFileSync, it's generally frowned upon, but in this case I absolutely need these files.  I don't think there's any benefit in loading these security async